networkd/network_server.ml

```diff
@@ -833,6 +859,16 @@ module Bridge = struct
 				(String.concat ", " (List.map (fun (name, _) -> name) config))
 				(if conservative then " (best effort)" else "");
 			List.iter (function (bridge_name, ({ports; vlan; bridge_mac; other_config; _} as c)) ->
+				(* Fail PIF.plug with exception Vlan_in_use if VLAN is in use by kernel and unknown to XAPI *)
+				begin match c.vlan with
+				| None -> ()
+				| Some (parent, id) ->
+					if Xstringext.String.has_substr dbg "PIF.plug" then begin
```

Comment:
We really can't do this... The `dbg` string must not be interpreted. And xcp-networkd should not need to know anything about `PIF.plug` or other xapi operations.